packages/nimble-components/src/table/index.ts

```diff
@@ -77,6 +71,11 @@ export class Table<
         _prev: string | undefined,
         _next: string | undefined
     ): void {
+        if (this.idFieldName == null) {
```

Comment:
Elsewhere in nimble we use explicit (val === null || val === undefined) checks. I think that is more explicit and forces considering values more carefully.

I'd prefer we do the explicit checks but I'm open to a follow-up change if the team prefers this syntax to do it more consistently in nimble (although I'd advocate for a `val == undefined` syntax instead to avoid proliferating `null` values in source).

Reply:
Sure, I can change this to be explicitly checking for null and for undefined.